pages/news/2020-06-07-mixxx-2-3-beta-released.html

```diff
@@ -0,0 +1,88 @@
+title: Mixxx 2.3 beta released
+author: Be.
+date: 2020-06-07 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+We are proud to announce the release of Mixxx 2.3 beta, [download it!](/download/) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more.
```

Comment:
```suggestion
We are proud to announce the release of Mixxx 2.3 beta, [download it!]({% url '/download.html' %}) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more.
```

Reply:
The link is working as it is. Why change it? IMO your suggested change just makes the code more verbose.

Reply:
Yes it works now, but that would change if we have to disable pretty links later. I think we should use `url` for links everywhere and you're already using `static` in this post.